### PR TITLE
[main] Matzew and vaikas dropping beta1

### PR DESCRIPTION
**Pull request by matzew**
contains parts of @vaikas  https://github.com/knative-sandbox/eventing-kafka/pull/501

and some clean ups ... :see_no_evil: :pray: 

/hold

### CODE REVIEW

**Comment by matzew** [comment]
@googlebot I consent.

File: test/upgrade/test_main.go [hunk 1]
```diff
@@ -24,14 +24,13 @@ import (
...
 	testlib "knative.dev/eventing/test/lib"
-	"knative.dev/eventing/test/lib/resources"
 	"knative.dev/pkg/system"
 )
 
 var (
 	channelTestRunner  testlib.ComponentsTestRunner
 	defaultChannelType = metav1.TypeMeta{
-		APIVersion: resources.MessagingAPIVersion,
+		APIVersion: "messaging.knative.dev/v1beta1",
```
Comment (matzew) [inline test/upgrade/test_main.go:33]: @cardil I am doing this here , now....

**Comment by matzew** [comment]
@googlebot I consent.

**Comment by vaikas** [comment]
@googlebot I consent.
/lgtm

**Review by vaikas — APPROVED** [review]

File: test/upgrade/test_main.go (hunk 1 above)
Comment (cardil) [reply test/upgrade/test_main.go:33]: Right, but should be better to have that in some variable?

**Pull request merged**